sqlglot/dialects.py

```diff
@@ -182,6 +182,14 @@ def _unnest_to_explode_sql(self, expression):
     return self.join_sql(expression)
 
 
+def _levenshtein_sql(self, expression, levenshtein_function):
```

Comment:
you don’t need this lambda

Reply:
Removed and replaced with some basic lambda funcs